reference/configuration/tidb-server/configuration-file.md

```diff
@@ -111,6 +111,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：[]
 + 默认情况下，该 list 名单为空，表示没有所需修复的坏表信息。
 
+### `max-server-connections`
+
++ TiDB 中同时允许的最大客户端连接数，用于资源控制。
++ 默认值：0
++ 默认情况下，TiDB 不限制客户端连接数。当本配置项的值大于 `0` 且客户端连接数到达此值时，TiDB 服务端将会拒绝新的客户端连接。
+
+### `max-index-length`
+
++ 用于设置新建索引的长度限制。
++ 默认值：3072
++ 单位：byte，目前的合法值范围 [3072, 3072*4]。MySQL 和 TiDB v3.0.11 之前版本没有此配置项，不过都对新建索引的长度做了限制。MySQL 对此的长度限制为 3072，TiDB 在 v3.0.7 以及之前版本此值为 3072*4。为了与 MySQL 和 TiDB 之前版本的兼容，添加了此配置项。
```

Comment:
v3.0.11 and v3.0.7, the versions seem inconsistent. If this is correct, I may wonder:
+ what's the length limit in versions between v3.0.7 and v3.0.11.
+ when this limit changed from 3072*4 to 3072.

Reply:
> what's the default length limit between v3.0.7 and v3.0.11.

3072.

> when this limit (default value) changed from 3072*4 to 3072.

3.0.11

Reply:
```suggestion
+ 单位：byte
+ 目前的合法值范围 `[3072, 3072*4]`。MySQL 和 TiDB v3.0.11 之前版本（不包含 v3.0.11）没有此配置项，不过都对新建索引的长度做了限制。MySQL 对此的长度限制为 `3072`，TiDB 在 v3.0.7 以及之前版本该值为 `3072*4`，在 v3.0.7 之后的该值为 `3072`。为了与 MySQL 和 TiDB 之前版本的兼容，添加了此配置项。
```

Reply:
@TomShawn Done.